frontend/src/hooks/useWords.js

```diff
@@ -5,14 +5,20 @@ export default function useWords() {
 
   const [allTypedInWords, setAllTypedInWords] = useState([]);
 
+  function doesWordExist(wordText) {
+    return axios
+        .get(`/api/word?wordText=${wordText}`)
+        .then((response) => response.data)
+  }
+
   function addWordToWordsList(wordText) {
     axios
-      .post("/api/letters", { wordText })
+      .post(`/api/word/`, { wordText })
       .then(() => setAllTypedInWords([...allTypedInWords, wordText]))
       .catch((error) => console.error(error.message));
```

Comment:
you want to consider using an error state which you can use to let your user know that adding the word has not been successful